pdns/dnsdist.cc

```diff
@@ -2639,6 +2647,10 @@ int main(int argc, char** argv)
       dropUserPrivs(newuid);
     }
 
+    if (retainedCapabilities) {
+      dropCapabilitiesAfterSwitchingIDs();
```

Comment:
what would be the harm in calling this in the `!retainedCapabilities` case?

Reply:
I don't see any from the top of my mind, but I prefer not to do anything at all when possible :)

Reply:
:+1: